templates/group/group.html

```diff
@@ -58,11 +58,18 @@ <h2>Members</h2>
 <h2>Meetings</h2>
 <ul>
     {% for meeting in meetings %}
+    {% match logged_in %}
+    {% when Some with (u) %}
     <li>
-        Meeting at {{ meeting.happened_at }} code:
-        <code>{{ meeting.code }}</code>
-        <a href="/big?text={{ meeting.code }}">View</a>
+        Meeting at {{ meeting.happened_at }}
+        {% if u.tier > 1 %}
```

Comment:
This should be > 0 since tier 1 is mentors who we do want to be able to view attendance codes